engine/install/index.md

```diff
@@ -146,9 +146,8 @@ Patch releases are always backward compatible with its year-month version.
 
 ### Licensing
 
-Docker is licensed under the Apache License, Version 2.0. See
-[LICENSE](https://github.com/moby/moby/blob/master/LICENSE) for the full
-license text.
+Docker Engine is licensed under the Docker Software End User License Agreement. See the
```

Comment:
To minimize confusion in the community, could you clarify that the source distribution of github.com/docker/{docker,cli,...} remains free under the terms of Apache License 2.0?

Reply:
Also curious, which version of Docker are covered by the Docker EULA?

Reply:
Hi, we are not changing the licenses of the sources at all. This PR is not worded in any way that is agreed or intended, so closing.